Buildings/Fluid/HeatExchangers/BaseClasses/WetCoilDryRegime.mo

```diff
@@ -0,0 +1,104 @@
+within Buildings.Fluid.HeatExchangers.BaseClasses;
+model WetCoilDryRegime
+  "Fully dry coil model"
+
+  // - water
+  input Modelica.SIunits.ThermalConductance UAWat
+    "UA for water side";
+  input Modelica.SIunits.MassFlowRate mWat_flow
+    "Mass flow rate for water";
+  input Modelica.SIunits.MassFlowRate mWatNonZer_flow(min=Modelica.Constants.eps)
```

Comment:
@kim1077 There are a lot of `min` attributes used for mass flow rate variables in this model and in `Buildings.Fluid.HeatExchangers.BaseClasses.WetCoilDryWetRegime`. 
How is that consistent with the support of flow reversal?
Why is the `min` attribute sometimes set to `Modelica.Constants.eps` and sometimes to 0?